internal/env.go

```diff
@@ -17,18 +18,20 @@ const (
 	DatadogAPMEndpoint = "DD_TRACE_AGENT_URL"
 	// DatadogEnableExtraProfiling is the environment variable key for whether to enable extra profiling or not.
 	DatadogEnableExtraProfiling = "DD_ENABLE_EXTRA_PROFILING"
+	// DatadogEnable is the environment variable key for whether to enable the Datadog integration.
+	DatadogEnable = "DD_ENABLE"
+	// DatadogDisable is the environment variable key for whether to disable the Datadog integration.
+	DatadogDisable = "DD_DISABLE"
 )
 
-// IsDatadogConfigured checks some common environment-variables to determine if the service is configured to use Datadog.
-func IsDatadogConfigured() bool {
-	if val := os.Getenv(DatadogEnvironment); val != "" {
-		return true
+// IsDatadogDisabled checks if the Datadog integration is disabled. The
+// environment variable DD_DISABLE is checked. If the variable is missing or
+// cannot be parsed to a bool the Datadog integration is assumed to be enabled.
+func IsDatadogDisabled() bool {
+	valStr := os.Getenv(DatadogDisable)
```

Comment:
Now it creates more confusion about how to use these env vars and functions... 

Suggesting to think about:
1. Use a single environment variable to control Datadog's enabled state. 
2. If disabling is rare, prefer checking DatadogEnable for clarity and simplicity. 
3. Remove or repurpose the unused DatadogDisable to avoid confusion.

Reply:
1. `DD_DISABLE` is now the only way to explicitly disable the library. Unless the consuming code uses other means to avoid enabling the library.
2. Every gRPC interceptor, database client, http client etc has to check if the library is disabled, so that is the common case
3. If you mean the const defining the environment variable name, that follows the pattern of the other environment variable names that the library knows about.

I do not understand how this creates more confusion, please elaborate.

Reply:
It was confusing before your commit (• ◡•) https://github.com/coopnorge/go-datadog-lib/pull/323/commits/8c77cc0f7795a498116bf820127618481b497cee